track_odometry/src/tf_projection_node.cpp

```diff
@@ -139,20 +142,24 @@ class TfProjectionNode
         parent_frame_);
 
     geometry_msgs::TransformStamped trans_out = tf2::toMsg(result);
-    if (flat_)
+    if (trans_out.header.stamp > prev_stamp_)
```

Comment:
It  will cause a problem on time jump back (especially on bag playback)

Reply:
Using `!=` instead of `>` could at least reduce the amount of warnings in regular use cases. Let me know if you think it would still cause some problems. If so I will just close this PR.